src/community/advanced-data-card.css.ts

```diff
@@ -0,0 +1,184 @@
+import {style, styleVariants} from '@vanilla-extract/css';
+import {sprinkles} from '../sprinkles.css';
+import * as mq from '../media-queries.css';
+import {vars} from '../skins/skin-contract.css';
+import {applyAlpha} from '../utils/color';
+
+export const actions = style([
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+        justifyContent: 'space-between',
+        paddingX: 16,
```

Comment:
paddingX in desktop views has to be setup in 24px to avoid this misalignment

![image](https://github.com/Telefonica/mistica-web/assets/6722153/0436d89b-0966-4ff2-b82e-f519af74ea87)